pantheon-decoupled.php

```diff
@@ -228,5 +238,46 @@ function pantheon_decoupled_test_preview_page() {
   <?php
 }
 
+function pantheon_decoupled_env_vars() {
+    if ( ! current_user_can( 'manage_options' ) ) {
+      wp_die( esc_html__( 'You do not have sufficient permissions to access this page.', 'wp-decoupled-preview' ) );
+    }
+    check_admin_referer( 'env-vars', 'nonce' );
+
+    // Get data for preview site
+    $id = isset( $_GET['id'] ) ? absint( sanitize_text_field( $_GET['id'] ) ) : NULL;
+    $preview_sites = get_option( 'preview_sites' );
+    $preview_site = isset( $preview_sites['preview'][ $id ] ) ? $preview_sites['preview'][ $id ] : NULL;
+      
+    ?>
+        <style>
+          /* Hide admin bar and padding on top of page. */
+          html.wp-toolbar {
```

Comment:
Note for future - see if we can enqueue these styles as a CSS file of some kind.